x/authz/client/testutil/tx.go

```diff
@@ -877,6 +918,86 @@ func (s *IntegrationTestSuite) TestNewExecGrantAuthorized() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestExecSendAuthzWithAllowList() {
```

Comment:
ref:https://github.com/cosmos/cosmos-sdk/pull/12648#discussion_r926256415 -> I added test for invalid address which is not in allow list here